src/applications/gi-sandbox/components/profile/EstimatedBenefits.jsx

```diff
@@ -92,25 +103,36 @@ export default function EstimatedBenefits({ profile, outputs, calculator }) {
 
       const learnMoreLink = `http://www.benefits.va.gov/gibill/comparison_tool/about_this_tool.asp#${section.toLowerCase()}`;
       const headerId = `${_.snakeCase(title)}_header`;
+
       return (
         <div key={section} className="per-term-section">
-          <div className="link-header">
-            <span id={headerId}>
-              <strong>{title}</strong>
-            </span>
-            <span className="vads-u-padding-left--2">
-              (
-              <a
-                href={learnMoreLink}
-                target="_blank"
-                rel="noopener noreferrer"
-                aria-label={learnMoreAriaLabel || ''}
-              >
-                Learn more
-              </a>
-              )
-            </span>
-          </div>
+          {outputs.perTerm[section].title === 'Housing allowance' && isOJT ? (
+            <LearnMoreLabel
+              text="Housing allowance"
+              onClick={() => dispatchShowModal('housingAllowanceOJT')}
+              ariaLabel="Learn more about how housing allowance is determined"
```

Comment:
Why is aria text different that what is at https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/gi-sandbox/constants.js#L85 ?

Reply:
Was going off of the text in the content doc for this one https://github.com/department-of-veterans-affairs/va.gov-team/blob/907b86789afadfe4b3db36a092fa66bc69cc9e5d/products/education-careers/school-comparison-tool/redesign/design/content-OJT.md#your-estimated-benefits